internal/cache/metadata/stat_cache.go

```diff
@@ -18,7 +18,6 @@ import (
 	"math"
 	"time"
 
-	"cloud.google.com/go/storage/control/apiv2/controlpb"
```

Comment:
Is the old one not used anymore?

Reply:
What about non HNS buckets? Would this not affect there.

Reply:
This is for folders only. We have created separate folder structure [here](https://github.com/GoogleCloudPlatform/gcsfuse/tree/master/internal/storage/gcs) for controlpb.folder to enable the addition of custom data in the future and to decouple folder struct in gcsfuse code from control package struct.

Reply:
yes this is just a small refactoring to have a our own struct for decoupling our methods from sdk folder